sourmash/_minhash.py

```diff
@@ -566,16 +566,19 @@ def is_molecule_type(self, molecule):
         """Check if this MinHash is a particular human-readable molecule type.
 
         Supports 'protein', 'dayhoff', 'hp', 'DNA'.
+        @CTB deprecate for 4.0?
```

Comment:
yes. See https://github.com/dib-lab/sourmash/issues/751 for longer discussion, especially https://github.com/dib-lab/sourmash/issues/751#issuecomment-547146368